src/scripts/dependencies/el6.sh

```diff
@@ -12,78 +12,28 @@
 #
 # Need to be root when running this script
 
-if [[ "$1" == "--install" ]]; then
-	echo "install"
+# worked on by Alex Rader, Cory Smith, Nicholas Robbins
 
-	yum -y clean all
-	yum -y clean expire-cache
+tinyosc_option="--tinyosc"
+ofx_option="--ofx"
 
-	# add epel and elrepo repos needed form some packages below
-	EL6TYPE=`head -1 /etc/issue | cut -d ' ' -f 1`
-	if [[ "$EL6TYPE" == "Scientific" ]];
-	then
-		yum install -y epel-release elrepo-release
-	else
-		# Presuming CentOS and RHEL
-		# From 'extras'
-		yum install -y epel-release
-		# From elrepo.org
-		rpm --import https://www.elrepo.org/RPM-GPG-KEY-elrepo.org
-		rpm -Uvh http://www.elrepo.org/elrepo-release-6-8.el6.elrepo.noarch.rpm
-	fi
+install_option="--install"
+cleanup_option="--cleanup"
+mode=0
 
-	# basic install/compile requirements
-	yum install -y git
+function install_tinyosc()
+{
 	yum install -y gcc
-	yum install -y make cmake
-	
-	# recent kernel install for the latest USB3 drivers
-	#yum --enablerepo=elrepo-kernel install -y kernel-ml-devel-4.8.7-1.el6.elrepo.x86_64
-	yum --enablerepo=elrepo-kernel install -y kernel-ml kernel-ml-devel
-
-	# OpenGL
-	yum install -y mesa-libGL
-	yum install -y mesa-libGL-devel
-
-	# TODO: refactor somehow; to select dynamically from lspci,
-	#       then download or dpkg-nvidia from elrepo
-	# TODO: this will need to be installed when booted
-	VIDEODRIVERSCRIPT=NVIDIA-Linux-x86_64-375.20.run
-	VIDEODRIVERPATH=XFree86/Linux-x86_64/375.20/$VIDEODRIVERSCRIPT
-	#VIDEODRIVER=XFree86/Linux-x86_64/340.104/NVIDIA-Linux-x86_64-340.104.run
-	wget us.download.nvidia.com/$VIDEODRIVERPATH
-	sh $VIDEODRIVERSCRIPT
-	
-	# install python 34 from epel
-	yum --enablerepo=epel install -y python34.x86_64
-
-	# packages for OpenGL and libusb
-	yum install -y libXmu-devel libXi-devel glut-devel libudev-devel libtool
-
-	# libfreenect2 dependencies
-	# libusb, requires libudev-devel, libtool from above
```

Comment:
Deletions should not have happened. The functionality during refactoring should always remain the same, but just the quality of the code should improve; loss of essential functionality is not refactoring.